### PR TITLE
Fix/button height

### PR DESCRIPTION
**Pull request by pedrohmorais**
## WHAT THIS PR DOES
Fixes the button component height when it have more than two lines.

## HOW TO REPRODUCE
Import the button component and force it to have two or more lines.

### CODE REVIEW

File: components/Button/Button.jsx [hunk 1]
```diff
@@ -59,10 +59,10 @@ const StyledButton = styled.button`
     },
   }) => {
     const paddings = {
```
Comment (alisson-suzigan) [inline components/Button/Button.jsx]: The papding values should not be multiples of two?

Comment (ggdaltoso) [reply components/Button/Button.jsx]: And it should be getting from the theme

Comment (pedrohmorais) [reply components/Button/Button.jsx]: A solution has been proposed in the last commit.

**Pull request merged**